boards/arm/nrf52_pca10040/board.h

```diff
@@ -41,4 +41,7 @@
 #define LED3_GPIO_PIN   20
 #define LED3_GPIO_PORT  CONFIG_GPIO_NRF5_P0_DEV_NAME
 
+/* GPIO pin pull up configuration */
+#define SW0_GPIO_PIN_PUD GPIO_PUD_PULL_UP
```

Comment:
Have it as below alongwith other group of SW0 defines:

```
#define SW0_GPIO_PIN	13
#define SW0_GPIO_PIN_PUD GPIO_PUD_PULL_UP
#define SW0_GPIO_NAME CONFIG_GPIO_NRF5_P0_DEV_NAME
```

Reply:
@cvinayak fixed